CONTRIBUTING.rst

```diff
@@ -0,0 +1,195 @@
+.. _dev-contributing:
+
+Contributing
+============
+
+By contributing to rascaline, you accept and agree to the following terms and
+conditions for your present and future contributions submitted to rascaline.
+Except for the license granted herein to rascaline and recipients of software
+distributed by rascaline, you reserve all right, title, and interest in and to
+your contributions.
+
+Code of Conduct
+---------------
+
+As contributors and maintainers of rascaline, we pledge to respect all people
+who contribute through reporting issues, posting feature requests, updating
+documentation, submitting merge requests or patches, and other activities.
+
+We are committed to making participation in this project a harassment-free
+experience for everyone, regardless of level of experience, gender, gender
+identity and expression, sexual orientation, disability, personal appearance,
+body size, race, ethnicity, age, or religion.
+
+Examples of unacceptable behavior by participants include the use of sexual
+language or imagery, derogatory comments or personal attacks, trolling, public
+or private harassment, insults, or other unprofessional conduct.
+
+Project maintainers have the right and responsibility to remove, edit, or reject
+comments, commits, code, wiki edits, issues, and other contributions that are
+not aligned to this Code of Conduct. Project maintainers who do not follow the
+Code of Conduct may be removed from the project team.
+
+This code of Conduct applies both within project spaces and in public spaces
+when an individual is representing the project or its community.
+
+.. Instances of abusive, harassing, or otherwise unacceptable behavior can be
+.. reported by emailing xxx@xxx.org.
+
+This Code of Conduct is adapted from the `Contributor Covenant`_, version 1.1.0,
+available at https://contributor-covenant.org/version/1/1/0/
+
+.. _`Contributor Covenant` : https://contributor-covenant.org
+
+Getting involved
+----------------
+
+Contribution via merge requests are always welcome. Source code is
+available from `Github`_. Before submitting a merge request, please
+open an issue to discuss your changes. Use the only `master` branch 
+for submitting your requests.
+
+Required tools
+--------------
+
+You will need to install and get familiar with the following tools when working
+on rascaline:
+
+- **git**: the software we use for version control of the source code. See
+  https://git-scm.com/downloads for installation instructions.
+- **the rust compiler**: you will need both ``rustc`` (the compiler) and
+  ``cargo`` (associated build tool). You can install both using `rustup`_, or
+  use a version provided by your operating system. We need at least Rust version
+  1.42.0 to build rascaline.
```

Comment:
This is a bit outdated, we now require at least version 1.56 (https://github.com/Luthaf/rascaline/blob/de7fb3508935b638166fd51c52f1689438c5cc30/.github/workflows/tests.yml#L34)